nautobot/utilities/templates/utilities/templatetags/filter_form_modal.html

```diff
@@ -0,0 +1,87 @@
+                            Default
+                        </a>
+                    </li>
+                    {% endif %}
+                    {% if dynamic_filter_form %}
+                    <li role="presentation" class="{% if not filter_form %} active {% endif %}">
+                        <a href="#advanced-filter" role="tab" data-toggle="tab">
+                            Advanced
+                        </a>
+                    </li>
+                    {% endif %}
+                </ul>
+                <div class="tab-content">
+                    {% if filter_form %}
+                    <div id="default-filter" role="tabpanel" class="tab-pane active">
+                        {% include 'inc/search_panel.html' %}
+                    </div>
+                    {% endif %}
+                    {% if dynamic_filter_form %}
+                    <div id="advanced-filter" role="tabpanel" class="tab-pane {% if not filter_form %} active {% endif %}">
+                        <form id="dynamic-filter-form" class="form-horizontal" method="get" action=".">
+                            <div class="">
+                                <table class="table" id="filterform-table">
+                                    {{ dynamic_filter_form.management_form }}
+                                    {% for form in dynamic_filter_form %}
+                                        {% if forloop.first %}
+                                            <thead>
+                                                <tr>
+                                                    {% for field in form.visible_fields %}
+                                                        <th>{{ field.label|capfirst }}</th>
+                                                    {% endfor %}
+                                                </tr>
+                                            </thead>
+                                        {% endif %}
+                                        <tr class="formset_row-dynamic-filterform">
+                                            {% for field in form.visible_fields %}
+                                                <td>
+                                                    {{ field }}
+                                                </td>
+                                            {% endfor %}
+                                        </tr>
+                                    {% endfor %}
+                                </table>
+                            </div>
+
+                            <div class="text-right noprint">
+                                <button type="submit" class="btn btn-primary">
+                                    <span class="mdi mdi-magnify" aria-hidden="true"></span> Apply
+                                </button>
+                                <a href="." class="btn btn-default">
+                                    <span class="mdi mdi-close-thick" aria-hidden="true"></span> Cancel
```

Comment:
This "default" tab button is labeled "Clear" and the dynamic tab button is labeled "Cancel", but they both in fact clear the search criteria. This was confusing for me.
```suggestion
                                    <span class="mdi mdi-close-thick" aria-hidden="true"></span> Clear
```